### PR TITLE
User authentication --front end

### PR DESCRIPTION
**Pull request by nagwan**

### CODE REVIEW

File: backend/controllers/usersController.js [hunk 1]
```diff
@@ -58,9 +58,9 @@ const registerUser = asyncHandler(async (req, res) => {
                 res.status(400)
                 throw new Error("invalid user data")
             }
-        }else {
+        } else {
             res.status(400)
-            throw new Error("invalid user data") 
+            throw new Error("invalid user data")
         }
     }
```
Comment (mohamedatef73) [inline backend/controllers/usersController.js:66]: here   
   const userExists = await User.findOne({ email }) 
i tried to find both of email and name using find method but it dosn't work 
So could i make this term or not ?

Comment (mohamedatef73) [inline backend/controllers/usersController.js:66]: And in this line 
const newUser = await User.create({....})
I want to create a file in the App inside back end server contain the new user's data 
Is it available ?

Comment (nagwan) [reply backend/controllers/usersController.js:66]: why do you want to do that?!

Comment (mohamedatef73) [reply backend/controllers/usersController.js:66]: Just want to see the user's data in my code

**Pull request merged**